contracts/libs/Ibc.sol

```diff
@@ -213,11 +217,16 @@ contract Ibc {
     function toStr(bytes32 b) public pure returns (string memory) {
         uint8 i = 0;
         while (i < 32 && b[i] != 0) {
-            i++;
+            unchecked {
+                ++i;
+            }
         }
         bytes memory bytesArray = new bytes(i);
-        for (uint8 j = 0; j < i; j++) {
+        for (uint8 j = 0; j < i;) {
             bytesArray[j] = b[j];
+            unchecked {
```

Comment:
I think this unchecked math is fine for now, though we might also want to look into using OZ's string util methods here as well

Reply:
The argument is that j is bounded by i which is bounded by 32.